### PR TITLE
enabling ACA random in HMatrix

### PR DESCRIPTION
**Pull request by poncetr**
In hmat-oss, compression method 4 is AcaRandom.
Please merge to be able to use this feature.

### CODE REVIEW

File: lib/src/Base/Stat/HMatrixParameters.cxx [hunk 1]
```diff
@@ -129,8 +132,10 @@ UnsignedInteger HMatrixParameters::getCompressionMethodAsUnsignedInteger() const
     return 2;
   else if (compressionMethod_ == "ACA+")
     return 3;
+  else if (compressionMethod_ == "ACA random")
```
Comment (jschueller) [inline lib/src/Base/Stat/HMatrixParameters.cxx]: you need to mention in the doc, see HMatrixParameters_doc.i.in

**Comment by jschueller** [comment]
which version of hmat has this ? master from 21st october ?
https://github.com/jeromerobert/hmat-oss/commit/bf70719e963a858de52dade5defb49ad02f94248

are there any plans for a future hmat-oss release ?

File: lib/src/Base/Stat/HMatrixParameters.cxx (hunk 1 above)
Comment (poncetr) [reply lib/src/Base/Stat/HMatrixParameters.cxx]: Added. Thank you.

**Comment by sofianehaddad** [comment]
> which version of hmat has this ? master from 21st october ?
> [jeromerobert/hmat-oss@bf70719](https://github.com/jeromerobert/hmat-oss/commit/bf70719e963a858de52dade5defb49ad02f94248)
> 
> are there any plans for a future hmat-oss release ?

@jschueller yes future release is coming soon.

**Comment by jschueller** [comment]
@sofianehaddad great!

**Pull request merged**